File: Assignment1/consSum.cpp
#include <iostream>
using namespace std;

int main() {
    int n, sum = 0;
    cout << "Enter number: ";
    cin >> n;
    while (n > 9) {
        int x = n % 100;
        sum += x;
        n /= 10;
    }
    cout << sum << endl;
    return 0;
}

File: Assignment1/consSum.java
import java.util.*;

public class consSum
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter number: ");
		int n = sc.nextInt();
		int sum = 0;
		while (n>9)
		{
			int x = n%100;
			sum += x;
			n /= 10;
		}
		System.out.println(sum);

		sc.close();
	}
}